tests/components/miele/__init__.py

```diff
@@ -11,3 +14,10 @@ async def setup_integration(hass: HomeAssistant, config_entry: MockConfigEntry)
 
     await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
+
+
+def get_callback(
+    mock: AsyncMock, the_callback: str
+) -> Callable[[int], Awaitable[None]]:
+    """Get registered callbacks for api push."""
+    return mock.listen_events.call_args_list[0].kwargs.get(the_callback)
```

Comment:
I would recommend to just duplicate this one so you don't have those awkward strings to get the right one

Reply:
What do you mean by duplicate? There is only one registered callback and it has two keyword arguments so the loop from niko_home_control is not really applicable.

Reply:
```py
    data_callback = get_callback(mock_miele_client, "data_callback")
    await data_callback(device_fixture)
    await hass.async_block_till_done()

    act_file = load_json_object_fixture("4_actions.json", DOMAIN)
    action_callback = get_callback(mock_miele_client, "actions_callback")
```

I rather would do
```py
    data_callback = get_data_callback(mock_miele_client)
    await data_callback(device_fixture)
    await hass.async_block_till_done()

    act_file = load_json_object_fixture("4_actions.json", DOMAIN)
    action_callback = get_actions_callback(mock_miele_client)
```